src/Codeception/Test/Feature/BlockByMetadata.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Codeception\Test\Feature;
+
+use Codeception\Test\Metadata;
+
+trait BlockByMetadata
```

Comment:
Or to choose a better name...

* SkipByMetadata
* IgnoreFromMetadata

I will think on it